### ISSUE TITLE
[SF_ACC]  ui5-input adding aria-expanded to input when suggestions are enable

### ISSUE THREAD

**Issue by lormanlau** [SAP/ui5-webcomponents#6039]
### **Feature Request Description**
A clear and concise description of what feature is missing.
Is your feature request related to a problem? Please describe.

a11y is asking for ui5-input to have the aria-expanded to be added when the suggestions are enabled

### **Proposed Solution**
A clear and concise description of the solution you would like to have.

In the shadowroot of the ui5-input, the input element needs to include the aria-expanded=true and aria-expanded=false while the suggestions are enabled.

### **Additional Context**
Add other context or screenshots related to the feature request here.

The ui5 web component already has something which adds the open attribute to the element, but the inner input element does not have the aria-expanded label

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

A clear and concise description of the impact/urgency of the required feature.

### **Stakeholder Info** (if applicable)

- Organization: SAP SuccessFactors
- Business impact: Medium

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

Could you evaluate this feature request.

Best Regards,
Boyan

**Comment by niyap** [comment]
Hello @lormanlau,

Thanks for sharing your findings!

According to the accessibility guidelines, it is not expected to have aria-expanded attribute set to the ui5-input component when suggestions are available. Here, you can find more information: https://www.w3.org/TR/wai-aria-1.2/#aria-expanded
Also, we are providing information to the screen reader users about the fact that there are suggestions available - when the user focus the ui5-input control "Suggestions Available" is announced and when the user types something "N results are available" is announced.

Kind Regards,
Niya
